src/config/Fees/index.ts

```diff
@@ -0,0 +1,36 @@
+import { FEES_42161 } from "./FEES_42161";
+import { FEES_43113 } from "./FEES_43113";
+import { ARBITRUM, ARBITRUM_TESTNET, AVALANCHE, MAINNET } from "../chains";
+
+const SECONDS_PER_WEEK = 604800;
+
+type FeeItem = {
+  from: number;
+  to: number;
+  feeUsd: string;
+};
+
+function createFeeList(data: { to: number; feeUsd: string }[]) {
+  const list: FeeItem[] = [];
+
+  for (let i = 0; i < data.length; i++) {
+    const item = data[i];
+    list.push({
+      from: item.to - SECONDS_PER_WEEK,
+      to: item.to,
+      feeUsd: item.feeUsd,
+    });
+  }
+  return list;
```

Comment:
Do you think `.map` will be a better way to write this? Maybe `.map` will make it more readable.